pkg/errors/errors.go

```diff
@@ -120,6 +120,9 @@ var (
 	ErrOldValueNotEnabled        = errors.Normalize("old value is not enabled", errors.RFCCodeText("CDC:ErrOldValueNotEnabled"))
 	ErrSinkInvalidConfig         = errors.Normalize("sink config invalid", errors.RFCCodeText("CDC:ErrSinkInvalidConfig"))
 	ErrCraftCodecInvalidData     = errors.Normalize("craft codec invalid data", errors.RFCCodeText("CDC:ErrCraftCodecInvalidData"))
+	// internal use only
```

Comment:
Why is it necessary to use two distinct errors?

Reply:
In current implementation 
1. when the caller meet a `ErrFlushTsBlocking` error, it can just ignore it, because the resolvedTs will continually send by the upstream, which is a implicit retry mechanism.
2. when the caller meet a `ErrRedoLogBufferBlocking` error, it should retry until it success, because the row event will send by the upstream only once.

Reply:
> In current implementation
> 
> 1. when the caller meet a `ErrFlushTsBlocking` error, it can just ignore it, because the resolvedTs will continually send by the upstream, which is a implicit retry mechanism.
> 2. when the caller meet a `ErrRedoLogBufferBlocking` error, it should retry until it success, because the row event will send by the upstream only once.

This seems to be a matter of the different semantics of different methods. But the underlying reasons for returning errors seem to be the same?